perf/prometheus-install/prometheus-operator-values.yaml

```diff
@@ -0,0 +1,1103 @@
+# Values for prometheus-operator.
```

Comment:
Was there a reason you copied all of this? You can just make a file with just the changes you apply, no need to have the defaults copied - it makes it impossible to tell what you changed

Reply:
Good point. Please make this change along with updating the the istio base prom config to be aware of operator in a subsequent PR.

Reply:
> /lgtm
> 
> We should templatize the istio standard prometheus config to support this so we don't include a special config here.

I agree that this should be the long term direction, but I am not yet confident enough of the operator to push it out so broadly.  My intent was to prove the operator here in the release qual testing before deploying it more fully, along with the install work @costinm is doing...